mava/advanced_usage/ff_ippo_store_experience.py

```diff
@@ -670,11 +670,12 @@ def _reshape_experience(experience: Dict[str, chex.Array]) -> Dict[str, chex.Arr
     logger.stop()
 
 
-@hydra.main(config_path="../configs", config_name="default_ff_ippo.yaml", version_base="1.2")
+@hydra.main(config_path="../configs/default", config_name="ff_ippo.yaml", version_base="1.2")
```

Comment:
default/  you need to add "/"

Reply:
It works with or without but nice catch it makes all files the same 🙌

Reply:
Well I guess I'll do the opposite and remove the '/' from all files 😅